components/layout/display_list/builder.rs

```diff
@@ -3151,22 +3151,22 @@ impl BaseFlowDisplayListBuilding for BaseFlow {
 }
 
 trait ComputedValuesCursorUtility {
-    fn get_cursor(&self, default_cursor: Cursor) -> Option<Cursor>;
+    fn get_cursor(&self, default_cursor: CursorKind) -> Option<CursorKind>;
 }
 
 impl ComputedValuesCursorUtility for ComputedValues {
     /// Gets the cursor to use given the specific ComputedValues.  `default_cursor` specifies
     /// the cursor to use if `cursor` is `auto`. Typically, this will be `PointerCursor`, but for
     /// text display items it may be `TextCursor` or `VerticalTextCursor`.
     #[inline]
-    fn get_cursor(&self, default_cursor: Cursor) -> Option<Cursor> {
+    fn get_cursor(&self, default_cursor: CursorKind) -> Option<CursorKind> {
         match (
-            self.get_pointing().pointer_events,
-            self.get_pointing().cursor,
+            &self.get_pointing().pointer_events,
+            &self.get_pointing().cursor,
         ) {
-            (PointerEvents::None, _) => None,
-            (PointerEvents::Auto, cursor::Keyword::Auto) => Some(default_cursor),
-            (PointerEvents::Auto, cursor::Keyword::Cursor(cursor)) => Some(cursor),
+            (&PointerEvents::None, _) => None,
+            (&PointerEvents::Auto, &Cursor(CursorKind::Auto)) => Some(default_cursor),
+            (&PointerEvents::Auto, &Cursor(cursor)) => Some(cursor),
```

Comment:
Sorry, I think I'm not familiar with borrow checker enough.
I'm just curious why we have to use reference here?
If we don't use it, the value will be moved?